tray/menu.go

```diff
@@ -0,0 +1,174 @@
+package tray
+
+import (
+	"os/exec"
+	"strings"
+
+	"github.com/NordSecurity/systray"
+)
+
+func addDebugSection() {
+	systray.AddSeparator()
+	mRedraw := systray.AddMenuItem("Redraw", "Redraw")
+	go func() {
+		for {
+			<-mRedraw.ClickedCh
+			redrawChan <- struct{}{}
+		}
+	}()
+	mUpdate := systray.AddMenuItem("Update", "Update")
+	go func() {
+		for {
+			<-mUpdate.ClickedCh
+			updateChan <- false
+		}
+	}()
+	mUpdateFull := systray.AddMenuItem("Full update", "Full update")
+	go func() {
+		for {
+			<-mUpdateFull.ClickedCh
+			updateChan <- true
+		}
+	}()
+}
+
+func addQuitItem() {
+	systray.AddSeparator()
+	m := systray.AddMenuItem("Quit", "Quit")
+	m.Enable()
+	go func() {
+		<-m.ClickedCh
+		systray.Quit()
+	}()
+}
+
+func addAppSection() {
+	m := systray.AddMenuItem("NordVPN", "NordVPN")
+	go func() {
+		for {
+			<-m.ClickedCh
+			cmd := exec.Command("zenity", "--info", "--text", "This will be the NordVPN GUI", "--no-wrap")
+			err := cmd.Start()
+			if err != nil {
+				notification("error", "Failed to start NordVPN GUI: %s", err)
+			} else {
+				err = cmd.Wait()
+			}
+		}
+	}()
+}
```

Comment:
Can be removed. This will require additional logic for displaying this anyways
```suggestion
```

Reply:
I think that we need to keep the "NordVPN" as the first button in the menu anyways. Otherwise user would have no indicator what application does that tray icon comes from. It may be doing nothing on click before GUI is launched.